### ISSUE TITLE
Absolute navigation started on popup page does not clear the navigation stack

### ISSUE THREAD

**Issue by TKNos** [dansiegel/Prism.Plugin.Popups#144]
I updated to the latest version **7.2.0.1046** to test the fix for #106. While I can confirm that issue #106 has indeed been resolved, I discovered a new issue:

1. Start at `ContentPageA`
2. Perform a relative navigation to `ContentPageB`
3. Perform a relative navigation to `PopupPage`
4. Perform an absolute navigation to `ContentPageC`

After step 4, `ContentPageC` is shown but the navigation stack was not cleared, i.e. using the back button navigates back to `ContentPageB`. In my opinion, the absolute navigation to `ContentPageC` should clear the stack.

**Update**: I created a short example reproducing the error. Hope it helps. Please see [Issue_144.zip](https://github.com/dansiegel/Prism.Plugin.Popups/files/4475517/Issue_144.zip)

**Comment by borninmusic** [comment]
Experiencing exactly the same issue. 
Rolling back to **7.2.0.759** fixes this behavior.

**Comment by studeruss** [comment]
We also experience the same issue

**Comment by vincooys** [comment]
Interestingly enough, this issue also exists when not navigating from a popup page. 

I updated Prism.Plugin.Popups to the latest version (7.2.0.1046), and when navigating like this from another navigation page:

_navigationService.NavigateAsync("/" + nameof(CustomTabbedPage), null, false, true);

...the navigation stack isn't cleared. Took me a while to figure out that it was the plugin popup that caused it, but rolling back to version 7.2.0.759 fixes the issue!

Just wanted to add this in case anyone else experiences the same issue and is wondering what is causing it!

**Comment by dansiegel** [comment]
This issue is now fixed and is available to sponsors in the SponsorConnect feed.
